modules/network-observability-operator-install.adoc

```diff
@@ -9,9 +9,10 @@ You can install the Network Observability Operator using the {product-title} web
 
 .Prerequisites
 
-* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
-* One of the following supported architectures is required: `amd64`, `ppc64le`, `arm64`, or `s390x`.
-* Any CPU supported by Red Hat Enterprise Linux (RHEL) 9
+* If you choose to use Loki, install the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7+].
+* One of the following supported architectures is required: amd64, ppc64le, arm64, or s390x.
+* Any CPU supported by Red Hat Enterprise Linux (RHEL) 9.
```

Comment:
can you explain what does this CPU bullet mean?

Reply:
Lol @memodi you're really testing my GitHub thread retrieval skillz 🤓 It was added in this PR with the multi-arch stuff:
https://github.com/openshift/openshift-docs/pull/59694#discussion_r1212674132

Reply:
and https://issues.redhat.com/browse/NETOBSERV-1067

Reply:
I think its showing up as "new" here because I had to rebase Dhruv's PR and pull this in, so it shows as a new add even though it was there in the past release.

Reply:
ah, okay. I was trying to make sense with some context, thanks for providing it and sorry about that :D

Reply:
NP, its kinda fun when I actually retrieve the thread I'm thinking of 🤓